src/css/global-dark.css

```diff
@@ -48,6 +48,9 @@
   input[type=time]::-webkit-calendar-picker-indicator {
     filter: invert(1);
   }
+  input[type="range"] {
```

Comment:
If this is Firefox-only, it should be moved to the end of the file after `/* Workarounds for FF bugs/quirks */`

Reply:
This is for chromium browsers to make the range track dark